internal/resources/resource_certificate_secret.go

```diff
@@ -67,16 +68,27 @@ func (r *certificateSecretResource) Schema(_ context.Context, _ resource.SchemaR
 				Description: "Name of the certificate",
 			},
 
-			"certificate": schema.StringAttribute{
+			"plaintext_value": schema.StringAttribute{
```

Comment:
I think this got mixed up - `plaintext_value` here has the description for the public key and `encrypted_value` the private key. Both values should represent the private key.

So I believe this should be;
- plaintext_value = plaintext private key
- encrypted_value = encrypted private key

And to reduce confusion, `certificate_value` should perhaps be `certificate_public` ? so that it's clear that's the public key - what's your thoughts on this?